pkg/registry/core/service/ipallocator/controller/repairip.go

```diff
@@ -212,6 +242,9 @@ func (r *RepairIPAddress) RunUntil(onFirstSuccess func(), stopCh chan struct{})
 	}
 	onFirstSuccess()
 
+	// cidrWorker locks so we don't win anything parallelizing
+	go wait.Until(r.cidrWorker, r.workerLoopPeriod, stopCh)
```

Comment:
"so we don't win anything by running multiple copies"?

You're still parallelizing it with ipWorker and svcWorker...

Reply:
yeah, but for handling the ServiceCIDR events we don't win anything running multiple workers, do we?

Reply:
I was just saying that the comment is confusing. You say that it doesn't make sense to run it in parallel, but then you run it in parallel (with ipWorker and svcWorker)...